server/api/api.go

```diff
@@ -3021,9 +3035,16 @@ func (a *API) handleDuplicateBlock(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !a.permissions.HasPermissionToBoard(userID, boardID, model.PermissionManageBoardCards) {
-		a.errorResponse(w, r.URL.Path, http.StatusForbidden, "", PermissionError{"access denied to modify board members"})
-		return
+	if block.Type == model.TypeComment {
```

Comment:
How would a user duplicate a comment unless they are duplicating a board or card?  Does this permission really apply to that case?

Reply:
From the API perspective, I think it doesn't matter, if you can duplicate a block, we should check that permission.